packages/composite-checkout/src/lib/line-items.js

```diff
@@ -25,3 +25,11 @@ export function useTotal() {
 	const [ , total ] = useLineItems();
 	return total;
 }
+
+export function useLineItemsOfType( itemType = null ) {
```

Comment:
I don't think we need to provide a default value of `null` if it's actually required (JS won't complain about missing arguments anyway).